jobs/postgres/templates/pre-start.erb

```diff
@@ -2,14 +2,14 @@
 
 set -eu
 
-PACKAGE_DIR=/var/vcap/packages/postgres-13
-PACKAGE_DIR_OLD=/var/vcap/packages/postgres-10
+PACKAGE_DIR=/var/vcap/packages/postgres-15
+PACKAGE_DIR_OLD=/var/vcap/packages/postgres-13
 
 PERSISTENT_DISK_DIR=/var/vcap/store
 
-STORE_DIR=${PERSISTENT_DISK_DIR}/postgres-13
-STORE_DIR_OLD=${PERSISTENT_DISK_DIR}/postgres-10
-STORE_DIR_OBSOLETE=${PERSISTENT_DISK_DIR}/postgres-9.4
```

Comment:
We are fully dropping detection of postgres-9.4 here.

The last version that shipped postgres 9 was 271.20, which was cut February of 2022. That version also had postgres 10, but the postgres 9 job was still there.

If somebody does upgrade from 271.20 or before with a postgres 9, it won't delete their data, but it will deploy successfully and come up with a totally empty director, and then probably take a bit of manual work to get themselves back into a happy state.

Not saying we must include a check for postgres 9 still, mostly just looking to get everybody's opinion on it.

Reply:
What about failing in the case of postgres 9 with an error message which suggest an update to an older version first and then to the ones which support postgres 15?

Reply:
@beyhan @jpalermo ive added changes to detect both 9.4 and 10 as obsolete versions and fail out